pages/blog/mistake-odyssey.md

```diff
@@ -0,0 +1,63 @@
+---
+title: "Mistake Odyssey"
+Subtitle: "Git, Github, VS Code and Markdown for non-programmers"
+date: 2021-07-08T06:00:00+01:00
+type: Communication
+tags:
+  - Git
+  - GitHub
+  - VS Code
+  - Markdown
+cover: img/posts/mistake-odyssey/mistake-odyssey-carolinie-cavalli.webp
```

Comment:
```suggestion
cover: /img/posts/mistake-odyssey/mistake-odyssey-carolinie-cavalli.webp
```

It's important that URLs paths are absolute (they start with `/`), otherwise, it means they are understood as relative to the current page. An example:

If the user is in https://www.asyncapi.com/blog, the current "path" is `/blog`. Therefore, the cover image would be resolved as `/img/posts/...`. However, if we then navigate to `https://www.asyncapi.com/blog/mistake-odyssey`, the current path would be `/blog/mistake-odyssey`, and the cover image would be resolved as `/blog/img/posts/...`. That would result in an "image not found" error. That's why you see it [here](https://deploy-preview-300--asyncapi-website.netlify.app/blog) but not [here](https://deploy-preview-300--asyncapi-website.netlify.app/blog/mistake-odyssey).